src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml

```diff
@@ -160,3 +160,26 @@
   tags:
     - formplayer_deploy
   when: not _should_update_formplayer_in_place
+
```

Comment:
I'm not confident in placing this at the end of the file due to the comment above the last task `Link formplayer jar`, but figure this is unrelated to a deploy?

Reply:
Hey @gherceg your logic seems sound. I think you should update the comment to be more specific/correct and say something more like
```
# This should be the last task in the formplayer release process
# (and the last one tagged with `formplayer_deploy`)
...
```